edo/.htaccess

```diff
@@ -0,0 +1,5 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Redirect www.w3id.org/edo to documentation
+RewriteRule ^edo(.*)$ https://www.mantech.com/edo$1 [R=302,L,NE]
```

Comment:
- Were you only wanting `/edo` to work?  Or did you want `/edo/foo` to go to `https://www.mantech.com/edo/foo`?
- `NE` flag is usually for when you have `#` in the target and don't want that escaped.  User agents won't send the hash and fragment to the server, so that probably not needed here.
-  If you want the main url with no slash, possible sub urls, and hash fragments are only on the user agents, try the below.
- Please test for your use cases.

```suggestion
RewriteRule ^$ https://www.mantech.com/edo [R=302,L]
RewriteRule ^(.*)$ https://www.mantech.com/edo/$1 [R=302,L]
```

- We try to avoid early claims on names for endpoints that don't yet exist.  Even those with the best of intentions may abandon projects and we'd like to avoid accumulating dead redirects.  Can you ping this issue when your site is live and we'll merge (once fixes are in too)?